src/Cms/LanguageRoutes.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+namespace Kirby\Cms;
+
+use Kirby\Toolkit\F;
+
+class LanguageRoutes
+{
+    /**
+     * Inject all language routes into
+     * the given routes array
+     *
+     * @param App $kirby
+     * @return array
+     */
+    public static function create(App $kirby): array
+    {
+        $routes[] = static::home($kirby);
```

Comment:
The `$routes` array is not initialized before.

Reply:
✅